spec/routes/web/project/postgres_spec.rb

```diff
@@ -404,6 +425,23 @@
         expect(page).to have_no_content "Danger Zone"
       end
 
+      it "allows updating initialization script when feature flag is enabled" do
+        project.set_ff_postgres_init_script(true)
+        visit "#{project.path}#{pg.path}/settings"
+        expect(page).to have_content "Initialization Script"
+        fill_in "init_script", with: "sudo whoami"
+
+        # We send PATCH request manually instead of just clicking to button because PATCH action triggered by JavaScript.
```

Comment:
We have a lot of these comments, and I don't think they add value. I also have plans to change things so we can use standard Capybara commands instead of `page.driver.submit` (already implemented in a few places, just not everywhere yet).

```suggestion
```